File: WorkoutTracker.Services/Models/WorkoutItem.cs
using System;

namespace WorkoutTracker.Core.Models
{
    public class WorkoutItem
    {
        public int Id { get; set; }
        public Exercise Exercise { get; set; }
        public int Reps { get; set; }
        public int Weight { get; set; }
        public DateTime AddDate { get; set; }
    }
}

File: WorkoutTracker.Angular/src/app/workoutitem/workoutitem.ts
import { Exercise } from '../exercise/exercise';

export interface WorkoutItem {
    WorkoutItemId: number;
    Exercise: Exercise;
    ExerciseId: number;
    Reps: number;
    Weight: number;
    WorkoutItemDate: Date;
    AddDate: Date;
    AddName: string;
}
